src/jit/codegeninterface.h

```diff
@@ -25,11 +25,14 @@
 #include "jitgcinfo.h"
 #include "treelifeupdater.h"
 
-// Disable this flag to avoid using psiScope/siScope info to report reporting
-// variables' home location during the method/prolog code.
 #if 1
+// Disable this flag to prevent using psiScope/siScope info to report variables' locations.
 #define USING_SCOPE_INFO
-#endif // USING_SCOPE_INFO
+#endif
+#if 1
+// Disable this flag to prevent using VariableLiveRange info to report variables' locations.
+#define USING_VARIABLE_LIVE_RANGE
+#endif
```

Comment:
I think this would be simpler to read as:

```
// Enable USING_SCOPE_INFO flag to use psiScope/siScope info to report variables' locations.
#define USING_SCOPE_INFO

// Enable USING_VARIABLE_LIVE_RANGE flag to use VariableLiveRange info to report variables' locations.
// Note: if both USING_SCOPE_INFO and USING_VARIABLE_LIVE_RANGE are defined, then USING_SCOPE_INFO
// information is reported to the debugger.
#define USING_VARIABLE_LIVE_RANGE
```